Updates to Teams calling policy pages (#9386)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md

### Identity (Default)
```
Set-CsTeamsCallingPolicy [[-Identity] <string>] [-AllowCallForwardingToPhone <bool>] [-AllowCallForwardingToUser <bool>] [-AllowCallGroups <bool>]
[-AllowCallRedirect <string>] [-AllowCloudRecordingForCalls <bool>] [-AllowDelegation <bool>] [-AllowNetworkConfigurationSettingsLookup <bool>]
[-AllowPrivateCalling <bool>] [-AllowSIPDevicesCalling <bool>] [-AllowTranscriptionForCalling <bool>] [-AllowVoicemail <string>]
[-AllowWebPSTNCalling <bool>] [-AutoAnswerEnabledType <string>] [-BusyOnBusyEnabledType <string>] [-CallRecordingExpirationDays <long>]
[-Description <string>] [-LiveCaptionsEnabledTypeForCalling <string>] [-MusicOnHoldEnabledType <string>] [-PreventTollBypass <bool>]
[-SafeTransferEnabled <string>] [-SpamFilteringEnabledType <string>] [-MsftInternalProcessingMode <string>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION
The Teams Calling Policy controls which calling and call forwarding features are available to users in Microsoft Teams. This cmdlet allows admins to set values in

> a given calling policy instance.

Reviewer: ```suggestion
a given Calling Policy instance.
```

PS C:\> Set-CsTeamsCallingPolicy -Identity Global -AllowPrivateCalling $true
```

> Sets the value of the parameter AllowPrivateCalling in the Global (default) Teams Calling policy instance.

Reviewer: ```suggestion
Sets the value of the parameter AllowPrivateCalling in the Global (default) Teams Calling Policy instance.
```

### Example 2
```
PS C:\> Set-CsTeamsCallingPolicy -Identity HRPolicy -LiveCaptionsEnabledTypeForCalling Disabled
```

> Sets the value of the parameter LiveCaptionsEnabledTypeForCalling to Disabled in the Teams Calling policy instance called HRPolicy.

Reviewer: ```suggestion
Sets the value of the parameter LiveCaptionsEnabledTypeForCalling to Disabled in the Teams Calling Policy instance called HRPolicy.
```

Accept wildcard characters: False
```

### -AllowCallGroups
Enables the user to configure call groups in the Microsoft Teams client and that inbound calls should be routed to call groups.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:
Applicable: Microsoft Teams

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowCallRedirect

> Setting this parameter provides the ability to configure call redirection capabilities on Teams Phones. The valid options are: Enabled, Disabled and UserOverride. When set to Enabled users will have the ability to redirect received calls. However, when set to Disabled the user will not have such ability. Note: UserOverride option is not available for use. There's no UX implemented for its management.

Reviewer: ```suggestion
Setting this parameter provides the ability to configure call redirection capabilities on Teams phones. The valid options are: Enabled, Disabled, and UserOverride. When set to Enabled users will have the ability to redirect received calls. However, when set to Disabled the user will not have such ability. Note: The UserOverride option is not available for use. There's no UX implemented for its management.
```

Accept wildcard characters: False
```

### -AllowCloudRecordingForCalls
Determines whether cloud recording is allowed in a user's 1:1 Teams or PSTN calls. Set this to True to allow the user to be able to record 1:1 calls. Set this to False to prohibit the user from recording 1:1 calls.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:
Applicable: Microsoft Teams

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowDelegation
Enables the user to configure delegation in the Microsoft Teams client and that inbound calls to be routed to delegates; allows delegates to make outbound calls on behalf of the users for whom they have delegated permissions.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:
Applicable: Microsoft Teams

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowNetworkConfigurationSettingsLookup

> This parameter is deprecated. Please use same parameter in the TeamsMeetingPolicy.

Reviewer: ```suggestion
This parameter is deprecated. Use the same parameter in the TeamsMeetingPolicy.
```

Accept wildcard characters: False
```

### -AllowPrivateCalling
Controls all calling capabilities in Teams. Turning this off will turn off all calling functionality in Teams.
If you use Skype for Business for calling, this policy will not affect calling functionality in Skype for Business.

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:
Applicable: Microsoft Teams

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowSIPDevicesCalling

> Determines whether the user is allowed to use SIP device for calling on behalf of teams client.

Reviewer: ```suggestion
Determines whether the user is allowed to use SIP devices for calling on behalf of a Teams client.
```

Accept wildcard characters: False
```

### -AllowVoicemail

> Enables inbound calls to be routed to voice mail. Valid options are: AlwaysEnabled, AlwaysDisabled, UserOverride. When set to AlwaysDisabled, calls are never routed to voicemail, regardless of the call forward or unanswered settings for the user. Voicemail isn't available as a call forwarding or unanswered setting in Teams. When set to AlwaysEnabled, calls are always forwarded to voicemail on unanswered after ringing for thirty seconds, regardless of the unanswered call forward setting for the user. When set to UserOverride, calls are forwarded to voicemail based on the call forwarding and/or unanswered settings for the user.

Reviewer: ```suggestion
Enables inbound calls to be routed to voicemail. Valid options are: AlwaysEnabled, AlwaysDisabled, and UserOverride. When set to AlwaysDisabled, calls are never routed to voicemail, regardless of the call forward or unanswered settings for the user. Voicemail isn't available as a call forwarding or unanswered setting in Teams. When set to AlwaysEnabled, calls are always forwarded to voicemail on unanswered after ringing for thirty seconds, regardless of the unanswered call forward setting for the user. When set to UserOverride, calls are forwarded to voicemail based on the call forwarding and/or unanswered settings for the user.
```

Accept wildcard characters: False
```

### -AllowWebPSTNCalling

> Allows PSTN calling from the Team web client.

Reviewer: ```suggestion
Allows PSTN calling from the Teams web client.
```

Accept wildcard characters: False
```

### -AutoAnswerEnabledType

> Setting this parameter allows you to enable or disable auto answer for incoming meeting invites on Teams Phones. It is turned off by default. Valid options are Enabled and Disabled. This setting applies only to incoming meeting invites and does not include support for other call types.

Reviewer: ```suggestion
Setting this parameter allows you to enable or disable auto-answer for incoming meeting invites on Teams phones. It is turned off by default. Valid options are Enabled and Disabled. This setting applies only to incoming meeting invites and does not include support for other call types.
```

```yaml
Type: Enum
Parameter Sets: (All)
Aliases:
Applicable: Microsoft Teams

Required: False
Position: Named
Default value: Disabled
Accept pipeline input: False
Accept wildcard characters: False
```

### -BusyOnBusyEnabledType

> Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: UserOverride option value is not available for use currently, if set it will be read as setting value to Disabled.

Reviewer: ```suggestion
Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, and Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: The UserOverride option value is not available for use currently, if set it will be read as setting the value to Disabled.
```

Accept wildcard characters: False
```

### -LiveCaptionsEnabledTypeForCalling

> Determines whether real-time captions are available for the user in Teams calls. Set this to DisabledUserOverride to allow user to turn on live captions. Set this to Disabled to prohibit.

Reviewer: ```suggestion
Determines whether real-time captions are available for the user in Teams calls. Set this to DisabledUserOverride to allow users to turn on live captions. Set this to Disabled to prohibit.
```

Accept wildcard characters: False
```

### -SpamFilteringEnabledType
Determines Spam filtering mode.

Possible values:

> - Enabled - Spam Filtering is fully enabled. Both Basic and Captcha IVR checks are performed. In case the call is considered as spam, user will get "Spam Likely" notification in Teams
> - Disabled - Spam Filtering is completely disabled. No checked are performed. "Spam Likely" notification will not appear.
> - EnabledWithoutIVR - Spam Filtering is enabled partially. Captcha IVR (Interactive Voice Response) checks are disabled. "Spam Likely" notification will appear. A call might get dropped in case it gets a high score from Basic checks.

Reviewer: ```suggestion
- Enabled: Spam Filtering is fully enabled. Both Basic and Captcha Interactive Voice Response (IVR) checks are performed. In case the call is considered as spam, the user will get a "Spam Likely" notification in Teams.
- Disabled - Spam Filtering is completely disabled. No checks are performed. A "Spam Likely" notification will not appear.
- EnabledWithoutIVR - Spam Filtering is partially enabled. Captcha IVR checks are disabled. A "Spam Likely" notification will appear. A call might get dropped if it gets a high score from Basic checks.
```
